### ISSUE TITLE
Is there a way to not have to authenticate this in browser every time?

### ISSUE THREAD

**Issue by Sarickshah** [panalysis/Google2Pandas#38]
I wanted to automate this so that it runs daily but dont want to have to authenticate it each time in browser. Is this possible?

**Comment by DeliciousHair** [comment]
Are you using the V3 or V4 api?

For V3, logging in via the browser downloads the analytics.dat file, and you need to change the permissions of it (`chmod 664 analytics.dat`) and simply point the class at it the next time you instantiate it as alluded to in the README.

For V4, I only used it the one time when I sketched up the class, but as I recall all that is done by adding the analytics account to the asset, and api access is then handled via the client_secrets.json file.

Bottom line, however, is that you shouldn't need to be logging in via the browser each time at all.
